package/MDAnalysis/core/topology.py

```diff
@@ -315,7 +315,7 @@ class Topology(object):
     """
 
     def __init__(self, n_atoms=1, n_res=1, n_seg=1,
-                 attrs=None,
+                 attrs=[],
                  atom_resindex=None,
```

Comment:
Have a look at the Quantified Code report. It flagged this issue *and* provides links to explanations.